workspace-manager-clienttests/src/main/java/scripts/testscripts/CreateGetDeleteControlledGcsBucket.java

```diff
@@ -63,7 +63,7 @@
                   .type(GcpGcsBucketLifecycleRuleActionType.SET_STORAGE_CLASS))
           .condition(
               new GcpGcsBucketLifecycleRuleCondition()
-                  .createdBefore(LocalDate.of(2017, 2, 18))
+                  .createdBefore(OffsetDateTime.parse("2007-01-03T00:00:00.00Z"))
```

Comment:
missing import? it looks like the client tests are failing here

Reply:
Yes. I'm too much of an optimist!